.zenodo.json

```diff
@@ -0,0 +1,75 @@
+{
+  "title": "MUSICA: Multi-Scale Infrastructure for Chemistry and Aerosols",
+  "description": "MUSICA is a multi-language library for performing computational simulations in atmospheric chemistry.",
+  "license": {
+    "id": "Apache-2.0"
+  },
+  "creators": [
+    {
+      "name": "Dawson, Matthew",
+      "affiliation": "National Center for Atmospheric Research (NCAR)",
+      "orcid": ""
+    },
+    {
+      "name": "Gim, Jiwon",
+      "affiliation": "National Center for Atmospheric Research (NCAR)",
+      "orcid": ""
+    },
+    {
+      "name": "Fillmore, David",
+      "affiliation": "National Center for Atmospheric Research (NCAR)",
+      "orcid": ""
+    },
+    {
+      "name": "Shores, Kyle",
+      "affiliation": "National Center for Atmospheric Research (NCAR)",
+      "orcid": ""
+    },
+    {
+      "name": "Thind, Montek",
+      "affiliation": "National Center for Atmospheric Research (NCAR)",
+      "orcid": ""
+    }
+  ],
+  "contributors": [
+    {
+      "name": "ACOM MUSICA Development Team",
+      "type": "ResearchGroup"
+    },
+    {
+      "name": "Angela Pak",
+      "type": "ProjectMember"
+    },
+    {
+      "name": "Aidan Winney",
+      "type": "ProjectMember"
+    },
+    {
+      "name": "Dee Grant",
+      "type": "Other"
+    },
+    {
+      "name": "Ward Fisher",
+      "type": "Other"
+    },
+    {
+      "name": "Jian Sun",
+      "type": "ProjectMember"
+    }
+  ],
```

Comment:
I think we could put Jian in with the creators, what do you think? Also, we should add Qina Tan here, since she wrote the majority of the CUDA solver code

Reply:
Fine with me